Meta/verify-sab-exposure.py

```
@@ -0,0 +1,604 @@
+                else:
+                    note = "[⚠️ no rejection emitted for this interface — how does the buffer arrive?]"
+                    ungated.append((path, n, line))
+            else:
+                note = "[ℹ️ no bindings layer]"
+            print(f"    {path}:{n}  {note}")
+            print(f"      {line[:96]}")
+        print()
+
+    if ungated:
+        print(f"⚠️ {len(ungated)} LibWeb call site(s) in an interface whose binding emits no rejection.")
+        print("   That's not automatically a bug; the interface *may* take no buffer-source parameter — such")
```

Ruff (0.16.2)

[warning] String contains ambiguous ` ` (NO-BREAK SPACE). Did you mean ` ` (SPACE)?

(RUF001)